litho-core/src/main/java/com/facebook/litho/InternalNode.java

```diff
@@ -1078,6 +1126,8 @@ public InternalNode backgroundAttr(@AttrRes int resId) {
 
   @Override
   public InternalNode backgroundRes(@DrawableRes int resId) {
+    checkIfBuilt();
+
```

Comment:
In some parts we have newline after `checkIfBuilt()` while in some others we don't. Can we make it consistent? I would prefer having newline after that check like here.